content/docs/04-clusters/1-new-clusters/01-aws.md

```diff
@@ -599,13 +599,270 @@ The following **four** policies include all the required permissions for provisi
+        "ec2:DescribeNetworkInterfaceAttribute",
+        "ec2:CreateSecurityGroup",
+        "ec2:ModifyVpcAttribute",
+        "iam:DeleteInstanceProfile",
+        "ec2:ReleaseAddress",
+        "iam:GetInstanceProfile",
+        "ec2:DescribeTags",
+        "ec2:DeleteRoute",
+        "ec2:DescribeNatGateways",
+        "ec2:DescribeIpamPools",
+        "ec2:AllocateAddress",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeImages",
+        "ec2:DescribeVpcs",
+        "elasticloadbalancing:DeleteLoadBalancer",
+        "elasticloadbalancing:DescribeLoadBalancers",
+        "elasticloadbalancing:DescribeLoadBalancerAttributes",
+        "elasticloadbalancing:CreateLoadBalancer",
+        "elasticloadbalancing:ModifyLoadBalancerAttributes",
+        "elasticloadbalancing:DescribeTags",
+        "secretsmanager:CreateSecret",
+        "secretsmanager:DeleteSecret",
+        "secretsmanager:TagResource",
+        "secretsmanager:GetSecretValue",
+        "autoscaling:StartInstanceRefresh",
+        "elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
+        "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+        "eks:DescribeCluster",
+        "eks:ListClusters",
+        "cloudformation:CreateStack",
+        "cloudformation:DescribeStacks",
+        "cloudformation:UpdateStack",
+        "ecr:GetAuthorizationToken",
+        "iam:PassRole",
+        "elasticloadbalancing:ConfigureHealthCheck",
+        "elasticloadbalancing:DescribeTargetHealth",
+        "ecr:BatchCheckLayerAvailability",
+        "ecr:GetDownloadUrlForLayer",
+        "ecr:GetRepositoryPolicy",
+        "ecr:DescribeRepositories",
+        "ecr:ListImages",
+        "ecr:BatchGetImage",
+        "ec2:DeleteInternetGateway",
+        "ec2:DeleteNatGateway",
+        "ec2:DeleteNetworkInterface",
+        "ec2:DeleteSecurityGroup",
+        "ec2:DeleteSubnet",
+        "ec2:DeleteTags",
+        "ssm:UpdateInstanceInformation",
+        "ssmmessages:CreateControlChannel",
+        "ssmmessages:CreateDataChannel",
+        "ssmmessages:OpenControlChannel",
+        "ssmmessages:OpenDataChannel",
+        "pricing:GetProducts",
+        "sts:AssumeRole",
+        "ec2:ReplaceRoute",
+        "ec2:ModifyNetworkInterfaceAttribute",
+        "ec2:AssociateAddress",
+        "tag:GetResources",
+        "ec2:ModifySubnetAttribute"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+         "iam:PassRole"
+      ],
+      "Resource": [
+          "arn:*:iam::*:role/*.cluster-api-provider-aws.sigs.k8s.io"
+      ]
+    }
+  ]
+}
+
+```
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Minimum Static Permissions" key="Minimum Static Permissions">
+
+<br />
+<br />
+
+This is a policy for use for those who want to restrict Palette to a single VPC and not give Palette access to create or delete VPCs.
+
+<br />
+<br />
+
+### Minimum Static Permissions
+
+
+```json
+{
+  "Version": "2012-10-17",

```

Comment:
Can we check with engineering to update a date - 2012 is 10 years ago

Reply:
This is fine. I think the schema version for AWS json/iam roles

Reply:
Thanks @sadysnaat for the confirmation.